products/jse_sub/features/jse_site_owner.feature

```diff
@@ -0,0 +1,124 @@
+@api
+Feature: Site Owner
+  In order to ensure that the Jumpstart Engineering administration functions are working properly
+  As a Site Owner
+  I want to check for my ability to complete my tasks
+
+
+  Background:
+      Given I am logged in as a user with the "site owner" role
+
+  @safe
+  Scenario: Can see Manage Content page
+    And I am on "admin/manage"
+    Then I should see 1 or more "tr" elements
+
+  @javascript @safe
+  Scenario: Clicking on Manage Content
+    When I wait for the Site Actions drop down to appear
+    And I click "Manage Content" in the "Admin Shortcuts" region
+    Then I should see 1 or more "tr" elements
+
+  @javascript @dev @destructive
+  Scenario: Add Stanford Page
+    When I wait for the Site Actions drop down to appear
+    And I click "Add Page" in the "Admin Shortcuts" region
+    Then I should see "Create Stanford Page" in the "Branding" region
+    And the "Text format" field should contain "content_editor_text_format"
+    When I click "Show Add/Edit Image"
+    And I wait 2 seconds
+    Then I should see "Source Info" in the "Content Body" region
+    Then I fill in "edit-title" with "Foo"
+    And I press the "Save" button
+    Then I should see "Stanford Page Foo has been created" in the "Console" region
+    And I should be on "foo"
+
+  @javascript @dev @destructive
+  Scenario: Add Landing Page
```

Comment:
This duplicates the tests in this file:
SU-SWS/permissions_tests/create_delete_landing_page/create_delete_landing_page.feature

Let's keep them in one place.